### DISCUSSION TITLE
Account Suspended for weeks without feedback

### DISCUSSION THREAD

**Discussion by ChillisoftsInferno** [community/community#141137] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My main account got suspended in early of September this year. I did have a hacker gain access to my computer but cleared it up within a day and spent an exuberant amount of time making sure all of my accounts we're reset. There were some services that suspended my account, but I was able to clear it up with them via support within a matter of days. 

I however did not receive an email from @GitHub as most other sources suggested would happen, and I have sent multiple requests and updates on the support ticket that was created. 

One post even went as far as to suggest patience when dealing with @GitHub's support... The problem is, my patience is weighing thin when no contact has been made with me to even attempt to sort this out from @GitHub's side and I have crucial work that I don't have local access to anymore, that I need to retrieve from my account.

Hopefully this will get your attention @GitHub and bring light to some other people who potentially, are having the same level of response or lack thereof.

Ticket: Reinstatement Request #2989752

**Comment by krishealty** [comment]
Hi there, the same happened to me few months ago where GitHub suspended my account for 4 weeks. The GitHub machine checks for any suspicious activity in your account and they just suspend it if they find any, without any email.

The reply email may delay after submitting a ticket due to rise in number of tickets every month. All you can do is wait and hope they reply to you. 

You can refer to [github docs](https://docs.github.com/en/enterprise-server@3.10/admin/managing-accounts-and-repositories/managing-users-in-your-enterprise/suspending-and-unsuspending-users) to check why your account is blocked.

**Reply by ChillisoftsInferno** [reply]
Thank you, I appreciate the feedback. I'm just hoping this post gets picked up by a mod. It's been over a month since my account got suspended.

**Reply by Warning39** [reply]
@krishealty did Github ever specify how you had violated their terms of service? Or did they simply undo the suspension 4 weeks later?

**Comment by LiteBrite82** [comment]
Hi @ChillisoftsInferno 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
